taipy/logger/_taipy_logger.py

```diff
@@ -34,7 +34,7 @@ def _get_logger(cls):
             cls.__logger.setLevel(logging.INFO)
             ch = logging.StreamHandler(sys.stdout)
             ch.setLevel(logging.INFO)
-            formatter = logging.Formatter("[%(asctime)s][%(name)s][%(levelname)s] %(message)s", "%Y-%m-%d %H:%M:%S")
-            ch.setFormatter(formatter)
+            f = logging.Formatter("[%(asctime)s.%(msecs)03d][%(name)s][%(levelname)s] %(message)s", "%Y-%m-%d %H:%M:%S")
```

Comment:
Add milliseconds to default logging format.

Reply:
Will this cause too much noise?

Reply:
My opinion is that in a multithreading/multiprocessing context, having milliseconds in the logs is more than useful to understand what's happening. This is not only true for us as developers but also for our users.

However, I agree we should discuss it during the daily to have more opinions on that.

Reply:
@FlorianJacta @FabienLelaquais Is it ok for you to have a default log formatting with milliseconds?